src/shared/components/query/QueryStore.ts

```diff
@@ -1790,7 +1790,8 @@ export class QueryStore {
             (!this.submitError &&
                 (this.genes.isComplete || this.genesets.isComplete) &&
                 this.asyncUrlParams.isComplete) ||
-            (!!this.oql.error || !!this.genesetIdsQuery.error)
```

Comment:
[Don't require parens for same-operator logical expressions](https://prettier.io/blog/2019/11/09/1.19.0.html#dont-require-parens-for-same-operator-logical-expressions-6864-by-jridgewell)